aries-site/src/themes/aries.js

```diff
@@ -20,11 +20,19 @@ export const aries = deepMerge(hpe, {
     background: 'background',
   },
   checkBox: {
+    color: 'selected-text',
+    gap: 'small',
     toggle: {
       color: {
         dark: hpe.global.colors['text-strong'].dark,
         light: hpe.global.colors['text-strong'].dark,
       },
+      check: {
```

Comment:
I think these changes should be happening just under checkBox as opposed to inside toggle

Reply:
good catch I didnt mean to have these in toggle